webrtc-stats.html

```diff
@@ -3362,6 +3363,16 @@ <h2>
                   [[!IANA-TLS-GROUPS]].
                 </p>
               </dd>
+              <dt>
+                <dfn><code>selectedCandidatePairChanges</code></dfn> of type <span class=
+                "idlMemberType">unsigned long</span>
+              </dt>
+              <dd>
+                <p>
+                  The number of times that the selected candidate pair of this transport has
+                  changed.
```

Comment:
Need to specify whether this is 0 or 1 after the initial selection of a candidate pair.

Reply:
Done